rest/diagnostic_doc_api_test.go

```diff
@@ -72,80 +72,18 @@ func TestGetDocDryRuns(t *testing.T) {
 	defer rt.Close()
 	bucket := rt.Bucket().GetName()
 	ImportFilter := `"function(doc) { if (doc.user.num) { return true; } else { return false; } }"`
-	SyncFn := `"function(doc) {channel(doc.channel); access(doc.accessUser, doc.accessChannel); role(doc.accessUser, doc.role); expiry(doc.expiry);}"`
-	resp := rt.SendAdminRequest("PUT", "/db/", fmt.Sprintf(
-		`{"bucket":"%s", "num_index_replicas": 0, "enable_shared_bucket_access": %t, "sync":%s, "import_filter":%s}`,
-		bucket, base.TestUseXattrs(), SyncFn, ImportFilter))
-	RequireStatus(t, resp, http.StatusCreated)
-	response := rt.SendDiagnosticRequest("GET", "/{{.keyspace}}/_sync", `{"accessChannel": ["dynamicChan5412"],"accessUser": "user","channel": ["dynamicChan222"],"expiry":10}`)
-	RequireStatus(t, response, http.StatusOK)
-	var respMap SyncFnDryRun
-	var respMapinit SyncFnDryRun
-	err := json.Unmarshal(response.BodyBytes(), &respMapinit)
-	assert.NoError(t, err)
-	assert.ElementsMatch(t, respMapinit.Exception, nil)
-	assert.Equal(t, respMapinit.Roles, channels.AccessMap{})
-	assert.Equal(t, respMapinit.Access, channels.AccessMap{"user": channels.BaseSetOf(t, "dynamicChan5412")})
-	assert.EqualValues(t, *respMapinit.Expiry, 10)
-	response = rt.SendDiagnosticRequest("GET", "/{{.keyspace}}/_sync", `{"role": ["role:role1"], "accessUser": "user"}`)
-	RequireStatus(t, response, http.StatusOK)
-
-	err = json.Unmarshal(response.BodyBytes(), &respMap)
-	assert.NoError(t, err)
-	assert.Equal(t, respMap.Roles, channels.AccessMap{"user": channels.BaseSetOf(t, "role1")})
 	newSyncFn := `"function(doc,oldDoc){if (doc.user.num >= 100) {channel(doc.channel);} else {throw({forbidden: 'user num too low'});}if (oldDoc){ console.log(oldDoc); if (oldDoc.user.num > doc.user.num) { access(oldDoc.user.name, doc.channel);} else {access(doc.user.name[0], doc.channel);}}}"`
-	resp = rt.SendAdminRequest("POST", "/db/_config", fmt.Sprintf(
+	resp := rt.SendAdminRequest("PUT", "/db/", fmt.Sprintf(
 		`{"bucket":"%s", "num_index_replicas": 0, "enable_shared_bucket_access": %t, "sync":%s, "import_filter":%s}`,
 		bucket, base.TestUseXattrs(), newSyncFn, ImportFilter))
 	RequireStatus(t, resp, http.StatusCreated)
```

Comment:
To work with collections see above for `RestTesterConfig` change.

```suggestion
resp := rt.CreateDatabase("db", rt.NewDbConfig())`
```